recirq/qml_lfe/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Insert qml_lfe on PYTHONPATH. This is to avoid the requirement
+# of needing to do from recirq.qml_lfe.xxx style imports inside of the
+# qml_lfe directory.
+import sys
+from pathlib import Path
+sys.path.insert(0, str(Path(__file__).parent))
```

Comment:
This seems suspicious.  I am not sure if the path trickery is worth it if it is just to make imports look nicer.

Reply:
I'm not a huge fan either, but I do think it is our best option. In certain cases if I called this module from different directories (other than directly inside qml_lfe) things might break. I'm also not a huge fan of doing `from recirq.qml_lfe.whatever import xxx` because it makes the code less standalone and self contained, which makes things harder than they need to be for a user who might want to pull this project out and hack on it.

Reply:
veto on this

Reply:
Alrighy, moved to using proper imports as pert style. It is a little akward launching off the modules now, but such is life.